tests/backends/test_dbmodel.py

```diff
@@ -398,7 +398,7 @@ def test_colocated_db_model_tf(fileutils, wlmutils, mlutils):
     test_script = fileutils.get_test_conf_path("run_tf_dbmodel_smartredis.py")
 
     # Create SmartSim Experience
-    exp = Experiment(exp_name, launcher=test_launcher)
+    exp = Experiment(exp_name, launcher=test_launcher, exp_path=test_dir)
```

Comment:
I will not address hypothetical issue in this file or in any file which is heavily touched by the Slurm test PR, as I think that will fix most of them